test/bin/start_api_test_server.sh

```diff
@@ -33,17 +33,70 @@ _get_osmt_project_dir() {
   echo "${project_dir}"
 }
 
-project_dir="$(_get_osmt_project_dir)" || exit 135
+get_spring_app_pid() {
```

Comment:
This function is not used in this script. Something similar is needed in the shutdown script.

Reply:
Removing it.